### ISSUE TITLE
fail to read Chinese words pdfs with decoding error

### ISSUE THREAD

**Issue by ChicoXYC** [hupili/python-for-data-and-media-communication-gitbook#135]
Target: We want to extract all the text from thousands of pdfs.

Problem: Decoding problem - Cannot read pdfs with Chinese words

Following are the trying examples we have made so far, but there is some encoding error, you can refer the following for details
http://nbviewer.jupyter.org/github/ChicoXYC/exercise/blob/master/get-text-from-pdf/read-chinese-pdf-with-encoding-error.ipynb

the pdfs can be found here: https://github.com/ChicoXYC/exercise/tree/master/get-text-from-pdf/pdfs

**Comment by lullabymia** [comment]
@hupili Could you please help us ?

**Comment by hupili** [comment]
may try other tools like “pandoc”

**Comment by lullabymia** [comment]
Does pandoc also accept files in commonmark, creole, docbook, docx, epub, fb2, gfm, haddock, html, jats, json, latex, man, markdown, markdown_github, markdown_mmd, markdown_phpextra, markdown_strict, mediawiki, muse, native, odt, opml, org, rst, t2t, textile, tikiwiki, twiki, vimwiki 
(no pdf)
<img width="1081" alt="2018-12-14 6 10 12" src="https://user-images.githubusercontent.com/42403858/49997046-89ca9880-ffcb-11e8-96b6-a93931dde426.png">
@hupili 
https://nbviewer.jupyter.org/github/lullabymia/example/blob/master/Untitled.ipynb

**Comment by ChicoXYC** [comment]
I‘ve already tried this method before, and tested it again today, didn't work out.

```
from pdfminer.pdfpage import PDFPage
```

error:
```python
 No module named 'pdfminer.pdfpage'
```
and 

```python
TypeError: __init__() got an unexpected keyword argument 'codec'
# didn't find the solution
```

**Comment by hupili** [comment]
how about importing pdfminer alone?

**Comment by ChicoXYC** [comment]
Yes, I tried this module alone. Same problem above. Didn't work out for Chinese

**Comment by hupili** [comment]
do you mean pdfminer works for English?

**Comment by ChicoXYC** [comment]
NO. english don't work either. It seems that the module function has been changed and I haven't found the answer. But pypdf2 works for English.
